src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -435,6 +435,12 @@
     <Field Name="GetTypeFromHandle" Type="MethodSymbol" Null="allow"/>
   </Node>
 
+  <Node Name="BoundMethodToken" Base="BoundExpression">
```

Comment:
This is a bound node that represents the metadata token value for a method. The strategy for finding span descriptors for particular methods at instrumentation time depends on tracking method tokens.